docs/modules/edged.md

```diff
@@ -0,0 +1,61 @@
+# EdgeD
+
+## Overview
+
+EdgeD is an edge node module which manages the POD lifecycle. With help of the POD specifications provided as JSON from Cloud via MetaManager, it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
+
+EdgeD interfaces with MetaManager module to receive all the configuration calls. It responds to MetaManager module with all config results and queries.
+
+Docker container runtime is used for container and image management. It is the default option for EdgeD. It is a configurable option in config.yaml.
```

Comment:
It is a configurable option in config.yaml -->
can we change to containerd ? or any other container runtime ?
what are the steps involved ? what is the effort ? what level of code changes are required ? 
we need to provide sufficient information to developers or users if we give a configurable option. Else we should document these details.